code/modules/mob/living/living.dm

```diff
@@ -382,6 +382,9 @@
 		if(!(L.status_flags & CANPUSH))
 			return
 
+		if(HAS_TRAIT(L, TRAIT_NO_PUSHING_BY_HUMAN) && istype(src, /mob/living/carbon/human))
```

Comment:
is type src. Beeeeh.

And putting something that will almost never happen in bump, that happens all the time, is so meh as well.

Find another solution that does not require to meddle with a proc like that

Reply:
i dont see why setting anchored wouldnt work